src/sentry/static/sentry/app/views/groupUserReports.jsx

```diff
@@ -107,10 +107,17 @@ const GroupUserReports = createReactClass({
         </div>
       );
     }
+
+    const emptyStateMessage = this.props.environment
+      ? tct('No user reports have been collected from your [env] environment.', {
```

Comment:
s/your/the/ ? not a strong opinion

Reply:
I like yours, but not a strong opinion, and not my words so will leave it :)